lib/versioning/nodever/index.js

```diff
@@ -0,0 +1,10 @@
+const semver = require('../semver');
+
+// Only LTS is stable
+const isStable = version =>
+  semver.isStable(version) && semver.matches(version, '^6.9.0 || ^8.9.0');
```

Comment:
Are we still waiting on https://github.com/nodejs/Release/issues/290 so that these hard-coded values can go away?

Reply:
Pretty much, yes. I think though that node.js plan far enough in advance so we could add conditional checks on dates too to remove the need for a commit/update on the exact days(s) that Node.js changes status.